Duckling/Numeral/CS/Corpus.hs

```diff
@@ -35,13 +35,51 @@ allExamples = concat
              , "jedno"
              ]
   , examples (NumeralValue 2)
-             [ "dva"
-             , "dvĕ"
+             [ "2"
+             , "dva"
+             , "dvě"
              ]
   , examples (NumeralValue 3)
-             [ "tři"
+             [ "3"
+             , "tři"
              ]
   , examples (NumeralValue 4)
-             [ "čtyři"
+             [ "4"
+             , "čtyři"
+             ]
+  , examples (NumeralValue 5)
+             [ "pět"
+             ]
+  , examples (NumeralValue 6)
+             [ "šest"
+             ]
+  , examples (NumeralValue 7)
+             [ "7"
+             , "sedm"
+             ]
+  , examples (NumeralValue 8)
+             [ "osm"
+             ]
+  , examples (NumeralValue 9)
+             [ "devět"
+             ]
+  , examples (NumeralValue 10)
+             [ "deset"
+             ]
+  , examples (NumeralValue 0.1)
```

Comment:
You need to update the rules to account for decimal numbers. You can look how it is handled in other languages.